### PR TITLE
zh-CN: Translate `lab()`, `lch()`, `oklab()`, `oklch()`

### PR DESCRIPTION
**Pull request by yarusome**
### Related issues and pull requests

### CODE REVIEW

File: files/zh-cn/web/css/color_value/lab/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("用_lab_调整明度和颜色轴")}}
+
+### 用 lab() 调整不透明度
+
+下列示例展示了改变函数记号 `lab()` 的 `A`（阿尔法）值的效果。为演示不透明度的效果， `red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```
Comment (yin1999) [inline files/zh-cn/web/css/color_value/lab/index.md:97]:
```suggestion
下列示例展示了改变函数记号 `lab()` 的 `A`（alpha）值的效果。为演示不透明度的效果，`red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```

File: files/zh-cn/web/css/color_value/lch/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("用_lch_调整明度、色度和色相")}}
+
+### 用 lch() 调整不透明度
+
+下列示例展示了改变函数记号 `lch()` 的 `A`（阿尔法）值的效果。为演示不透明度的效果， `red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```
Comment (yin1999) [inline files/zh-cn/web/css/color_value/lch/index.md:97]:
```suggestion
下列示例展示了改变函数记号 `lch()` 的 `A`（alpha）值的效果。为演示不透明度的效果，`red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```

File: files/zh-cn/web/css/color_value/oklab/index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("用_oklab_调整明度和轴", "100%", 155)}}
+
+### 用 oklab() 调整不透明度
+
+下列示例展示了改变 `oklab()` 函数的 `A`（阿尔法）值的效果。为演示不透明度的效果， `red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。给予 `red-alpha` 元素 `0.4` 的不透明度使其显得比 `red` 元素更透明。
```
Comment (yin1999) [inline files/zh-cn/web/css/color_value/oklab/index.md:104]:
```suggestion
下列示例展示了改变 `oklab()` 函数的 `A`（alpha）值的效果。为演示不透明度的效果，`red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。给予 `red-alpha` 元素 `0.4` 的不透明度使其显得比 `red` 元素更透明。
```

File: files/zh-cn/web/css/color_value/oklch/index.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("调整颜色的明度、色度和色相")}}
+
+### 调整颜色的阿尔法值
+
+下列示例展示了改变颜色函数 `oklch()` 的 `A`（阿尔法）值的效果。为演示不透明度的效果， `red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```
Comment (yin1999) [inline files/zh-cn/web/css/color_value/oklch/index.md:95]:
```suggestion
下列示例展示了改变颜色函数 `oklch()` 的 `A`（alpha）值的效果。为演示不透明度的效果，`red` 和 `red-alpha` 元素与 `#background-div` 元素重叠。`0.4` 的 `A` 值使颜色有 40% 的不透明度。
```

**Review by yin1999 — COMMENTED** [review]
考虑一下要不要翻译 alph

File: files/zh-cn/web/css/color_value/lab/index.md (hunk 1 above)
Comment (yarusome) [reply files/zh-cn/web/css/color_value/lab/index.md:97]: 有个问题：alpha 是否需要大写首字母？Photoshop 指南的[英文版](https://helpx.adobe.com/cn/photoshop/using/saving-selections-alpha-channel-masks.chromeless.html)用的 alpha，[中文版](https://helpx.adobe.com/cn/photoshop/using/saving-selections-alpha-channel-masks.chromeless.html)用的 Alpha。

Comment (yin1999) [reply files/zh-cn/web/css/color_value/lab/index.md:97]: 小写吧

**Review by yin1999 — APPROVED** [review]

**Pull request merged**